ss58-registry.json

```diff
@@ -211,6 +211,24 @@
     {
       "prefix": 22,
       "network": "dock-mainnet",
+      "displayName": "Dock POA Mainnet",
+      "symbols": ["DCK"],
+      "decimals": [6],
+      "standardAccount": "*25519",
+      "website": "https://dock.io"
+    },
+    {
+      "prefix": 21,
+      "network": "dock-pos-testnet",
+      "displayName": "Dock Testnet",
+      "symbols": ["DCK"],
+      "decimals": [6],
+      "standardAccount": "*25519",
+      "website": "https://dock.io"
+    },
+    {
+      "prefix": 22,
```

Comment:
@bkchr thoughts?

Reply:
it would be preferred for us to remove the old PoA information here so its less confusing to users providing there will be no issues downstream

Reply:
You need to remove the old information for sure. However, this requires bumping the crate to 2.0.0. As the name of the enum variants will change.

Or we add support for renaming and putting a deprecated above the old variants.

Reply:
Or you are okay with naming it `dock-mainnet` or `dock-testnet` as before.

Reply:
unfortunately cant rename to that, we had to add `-pos` in specname for various reasons (mainly, avoiding conflcits when upgrading). we may have people using the old spec name (dock-mainnet) to store old accounts in the polkadot-js extension or web app. ths spec name `dock-pos-xxnet` has been in use for a little bit now, so changing it would be very difficult or break things for users of polkadot apps/extension

bumping crate or support for deprecated sounds like best approach, ill let you decide which to take